exports.go

```diff
@@ -210,7 +210,13 @@ func (b *baseExportImpl) GenerateActivation(account string, issuer string) (stri
 		return "", err
 	}
 	ac := jwt.NewActivationClaims(key.Public)
-	ac.ImportSubject = b.export.Subject
+
+	if subject == "" {
```

Comment:
this is probably not a good idea, should probably error if not given, wdyt @aricart ?